Pod/Classes/PINAnimatedImageManager.m

```diff
@@ -214,16 +216,54 @@ - (void)animatedPathForImageData:(NSData *)animatedImageData infoCompletion:(PIN
   }
 }
 
+#define HANDLE_PROCESSING_ERROR(ERROR) \
```

Comment:
They're only relevant to the below method. I don't like that they're macros at all but couldn't think of a better way to do this :/